test/e2e_node/e2e_node_suite_test.go

```diff
@@ -172,12 +186,11 @@ func validateSystem() error {
 	if err != nil {
 		return fmt.Errorf("can't get current binary: %v", err)
 	}
-	// TODO(random-liu): Remove sudo in containerize PR.
-	output, err := exec.Command("sudo", testBin, "--system-validate-mode").CombinedOutput()
+	output, err := exec.Command(testBin, append([]string{"--system-validate-mode"}, os.Args[1:]...)...).CombinedOutput()
```

Comment:
Yeah, I want to make sure that all processes see the same flag set, or else it confuses people sometimes.